test/types/index.ts

```diff
@@ -609,13 +610,13 @@ server.listen({
 }).then((address: string) => console.log(address))
 
 // http injections
-server.inject({ url: '/test' }, (err: Error, res: fastify.HTTPInjectResponse) => {
+server.inject({ url: '/test' }, (err: Error, res: LightMyRequest.Response) => {
   server.log.debug(err)
   server.log.debug(res.payload)
 })
 
 server.inject({ url: '/testAgain' })
-  .then((res: fastify.HTTPInjectResponse) => console.log(res.payload))
+  .then((res: LightMyRequest.Response) => console.log(res.payload))
```

Comment:
Can you add another test with the old types as well?

Reply:
oops, I forgot it, sorry.